### ISSUE TITLE
API

### ISSUE THREAD

**Issue by zelix25** [pooler/cpuminer#277]
Hello, 
thank you for your fork, the software works relatively well.
Is it possible to have an API to exploit the miner's statistics?

THANKS

**Comment by hjafari7573** [comment]
Yes, dear friend, its price is 20ustd, send it to your wallet address:
TVeREZV8pZbqaDXxwc52Zq17aGWKrqWm9v

در تاریخ چهارشنبه ۲۰ دسامبر ۲۰۲۳،‏ ۲۱:۳۰ zelix25 ***@***.***>
نوشت:
